.github/workflows/reusable-docker-scan.yml

```
@@ -0,0 +1,302 @@
+          push: true
+          tags: ${{ steps.set-ghcr-ref.outputs.ghcr_image }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+  # ============================================
+  # VULNERABILITY SCAN (REUSABLE WORKFLOW)
+  # ============================================
+  vulnerability-scan:
+    needs: build-ghcr
+    name: Trivy Scan ${{ inputs.image_name }}${{ inputs.suffix }}
+    uses: liquibase/build-logic/.github/workflows/reusable-vulnerability-scan.yml@${{ inputs.build_logic_ref }}
```

actionlint (1.7.12)

[error] context "inputs" is not allowed here. no context is available here. see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)